pkg/gui/views.go

```diff
@@ -171,6 +171,7 @@ func (gui *Gui) createAllViews() error {
 	gui.Views.Extras.Title = gui.c.Tr.CommandLog
 	gui.Views.Extras.Autoscroll = true
 	gui.Views.Extras.Wrap = true
+	gui.Views.Extras.AutoRenderHyperLinks = true
```

Comment:
Unlike in Aleksei's earlier PR, I decided to not set the UnderlineHyperLinksOnlyOnHover in this view, following the earlier decision to turn this on only in the main view, but underline links directly everywhere else. Let me know if you dislike this @jesseduffield.

Reply:
That's fine.